### PR TITLE
add totalDifficulty to GetBlockByTimestamp

### PR DESCRIPTION
**Pull request by lupin012**
Added totalDifficulty to the returned response to be similar to the data returned by eth_getBlockByNumber() or eth_getBlockByHash()

### CODE REVIEW

**Comment by AskAlexSharov** [comment]
```
make lintci-deps
make lint
```

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-GO = go # if using docker, should not need to be installed/linked
+GO = /usr/local/go/bin/go
```
Comment (AskAlexSharov) [inline Makefile:1]: i absolutely not sure it's cross-platform

File: cmd/rpcdaemon/commands/erigon_block.go [hunk 2]
```diff
@@ -164,6 +164,18 @@ func buildBlockResponse(db kv.Tx, blockNum uint64, fullTx bool) (map[string]inte
 		return nil, nil
 	}
 
+<<<<<<< HEAD
```
Comment (AskAlexSharov) [inline cmd/rpcdaemon/commands/erigon_block.go:167]: please resolve conflict

**Review by AskAlexSharov — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (lupin012) [reply Makefile]: Sorry this my mistake I have commit a  local file.

Comment (lupin012) [reply Makefile]: Removed the mistake

**Review by AskAlexSharov — APPROVED** [review]

**Pull request merged**